Explorer/Assets/DCL/Chat/MessageBus/MultiplayerChatMessagesBus.cs

```diff
@@ -80,7 +80,7 @@ private async UniTaskVoid OnChatAsync(ReceivedMessage<Decentraland.Kernel.Comms.
             }
         }
 
-        private bool IsMention(string chatMessage, string userName)
+        private static bool IsMention(string chatMessage, string userName)
```

Comment:
Why this `static` addition?

Reply:
because why not :) just tiny optimization.